keyboards/40percentclub/gherkin/keymaps/midi/rules.mk

```diff
@@ -0,0 +1,75 @@
+# MCU name
```

Comment:
The keymap `rules.mk` should only be overrides, so the whole file should be something like
```
BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
MOUSEKEY_ENABLE = no	  # Mouse keys(+4700)
EXTRAKEY_ENABLE = no	  # Audio control and System control(+450)
CONSOLE_ENABLE = no	    # Console for debug(+400)
COMMAND_ENABLE = no     # Commands for debug and configuration
BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no
MIDI_ENABLE = yes
```